source/lib-intro.tex

```diff
@@ -1295,7 +1284,7 @@
 \ref{support.rtti}  & Type identification          &   \tcode{<typeinfo>}      \\ \rowsep
 \ref{support.exception} & Exception handling      &   \tcode{<exception>}     \\ \rowsep
 \ref{support.initlist}  & Initializer lists & \tcode{<initializer_list>} \\ \rowsep
-\ref{support.runtime} & Other runtime support     &   \tcode{<cstdalign>} \tcode{<cstdarg>} \tcode{<cstdbool>}       \\ \rowsep
```

Comment:
If everything else is now in Annex D, I would suggest the "add this to freestanding" text belongs in Annex D as well.